vowpalwabbit/core/src/reductions/epsilon_decay.cc

```diff
@@ -408,7 +410,7 @@ VW::LEARNER::base_learner* VW::reductions::epsilon_decay_setup(VW::setup_base_i&
                         .set_params_per_weight(model_count)
                         .set_output_prediction_type(base_learner->get_output_prediction_type())
                         .set_save_load(save_load_epsilon_decay)
-                        .set_finish(::finish)
+                        .set_output_example_prediction(output_example_epsilon_decay)
```

Comment:
Epsilon decay also needs to implement stats and update printing? Unless it can safely delegate to a lower reduction?

Reply:
safe to delegate to lower reductions